Rocket.Chat/API/Requests/General/PermissionsRequest.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PermissionsRequest.swift
+//  Rocket.Chat
+//
+//  Created by Matheus Cardoso on 5/1/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import SwiftyJSON
+
+class PermissionsRequest: APIRequest {
+    typealias APIResourceType = PermissionsResource
+
+    let requiredVersion: Version = Version(0, 61, 0)
+    let path = "/api/v1/permissions"
+}
+
+class PermissionsResource: APIResource {
+    var permissions: [Permission] {
+        return raw?.arrayValue.map {
+            let permission = Permission()
+            permission.map($0, realm: nil)
```

Comment:
Realm nil? 🤔

Reply:
We do the same thing in many places. We don't need the realm reference in Permission's implementation of ModelMappeable. 🙂